README.md

```diff
@@ -3,9 +3,16 @@ bsnes
 
 ![bsnes logo](bsnes/target-bsnes/resource/logo.png)
 
-bsnes is a multi-platform Super Nintendo (Super Famicom) emulator, originally
+**bsnes** is a multi-platform Super Nintendo (Super Famicom) emulator, originally
 developed by [Near](https://near.sh), which focuses on performance,
-features, and ease of use.
+features and ease of use.
```

Comment:
Not a fan of the oxford comma?

Reply:
It was just my auto-corrector complaining lol